packages/pwa-kit-dev/src/configs/webpack/config.js

```diff
@@ -498,8 +498,13 @@ const ssr = (() => {
     if (mode === production) {
         return baseConfig('node')
             .extend((config) => {
+                let additionalConfig = {}
+                if (process.env.SSR_SOURCE_MAP === 'true') {
```

Comment:
```suggestion
                if (process.env.PWA_KIT_SSR_SOURCE_MAP === 'true') {
```
Namespacing env vars is good/consistent 🙂